tools/qontract_cli.py

```diff
@@ -385,25 +385,42 @@ def soaking_str(soaking, upgrade_policy, upgradeable_version):
 for that cluster. The number in parenthesis shows the number of days this
 version has been running on other clusters with the same workloads. By
 comparing with the `soak_days` columns, you can see when a version is close to
-be upgraded to. A :tada: sign is displayed for versions which have soaked
-enough and are ready to be upgraded to. A :dizzy: sign is displayed for versions
+be upgraded to. A 🎉 sign is displayed for versions which have soaked
+enough and are ready to be upgraded to. A 💫 sign is displayed for versions
 which are scheduled or being upgraded to.
         """
         )
-
-    columns = [
-        "cluster",
-        "version",
-        "channel",
-        "schedule",
-        "mutexes",
-        "soak_days",
-        "workload",
-        "soaking_upgrades",
-    ]
-    ctx.obj["options"]["to_string"] = True
-    print_output(ctx.obj["options"], results, columns)
-
+        print("```json:table")
+        fields = [
+            {"key": "cluster", "sortable": True},
+            {"key": "version", "sortable": True},
+            {"key": "channel", "sortable": True},
+            {"key": "schedule"},
+            {"key": "mutexes", "sortable": True},
+            {"key": "soak_days", "sortable": True},
+            {"key": "workload"},
+            {"key": "soaking_upgrades"},
+        ]
+        print(
+            json.dumps(
+                {"fields": fields, "items": results, "filter": True, "caption": ""},
+                indent=2,
+            )
+        )
+        print("```")
```

Comment:
nit: construct a string and do one print only

Reply:
why is that better ?
We also have the explanatory text above.

Reply:
done

Reply:
looks better indeed :)

Reply:
it was just a nit. personally i try to avoid multiple prints if the output should be considered as one unit. i did not want to force that style on you. thank you for addressing it ❤️